src/app/find-help/transport-search/transport-search.component.html

```diff
@@ -4,7 +4,11 @@
 <app-no-results *ngIf="total === 0"></app-no-results>
 <div class="results" *ngIf="this.total" [class.loading]="loading">
   <router-outlet></router-outlet>
-  <div class="total">Lista ogłoszeń ({{ total }}):</div>
+  <app-sorting-header
+    [total]="total"
+    [(sortingOrder)]="modifiedDateSortOrder"
```

Comment:
I cannot se what modifies `modifiedDateSortOrder` from TransportSearchComponent, if non you could pass $event to search($event) only.